File: 파이썬베이스/정규표현식/반복/정확한구간찾기.py
import re


text = "body {background-color: #fefbd8;}h1 {background-color: #0000ff;}div {background-color: #d0f4e6;}span {background-color: #f08970;}"

# 방법 1
print(re.findall(r"#[A-Fa-f0-9]{6}", text))

File: 자스베이스/정규표현식/반복/정확한구간찾기.js
const text = `
    body {
        background-color: #fefbd8;
    } h1 {
        background-color: #0000ff;
    } div {
        background-color: #d0f4e6;
    } span {
        background-color: #f08970;
    }
`;

// 방법 1
// [ '#fefbd8', '#0000ff', '#d0f4e6', '#f08970' ]
console.log(text.match(/#[A-Fa-f0-9]{6}/g));
